notebooks/Notebook2.md

```diff
@@ -18,17 +20,41 @@ exciting milestones, preliminary results, etc. Feel free to include images
 code, and any other resources that you think will help clearly convey your
 design process.
 
+This week I mainly have been working on designing my language in Scastie. I have been trying to set up sbt on GitHub codespaces, but have ran into a lot of issues with that, so I have been using Scastie to test things out while I set that up. I have been playing around with the classes for different types of food entries. Each food has an assigned amount of calories that can be returned. Ideally the foods can also be grouped together into meals. For example, let's say that pizza is 200 calories and soda is 100. To log this as a meal you would enter:
+
```

Comment:
I can help with sbt, etc.! Let me know if you want to chat during studio time on Wednesday, or we can talk during office hours.